app/Controllers/Http/SearchController.ts

```diff
@@ -0,0 +1,47 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Definition from 'App/Models/Definition'
+
+export default class SearchController {
+  public async index({ request, response }: HttpContextContract) {
+    const { char, term } = request.qs()
+    const formattedTerm = decodeURI(term).trim()
+
+    if (!char || !formattedTerm) {
```

Comment:
This line of code will make the query `term` never execute, change OR conditional to AND (&&) conditional